### PR TITLE
update scripts to push to nexus v1 prod

### PR DESCRIPTION
**Pull request by kenjinp**

### CODE REVIEW

File: scripts/neuron-pipe/consts.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ const resourceDict = {
     source: "BBP NSE"
   },
   pctc: {
-    url: "https://bbp-nexus.epfl.ch/staging/v0/data/somatosensorycortexproject/electrophysiology/experimentaltraceset/v0.0.1/",
+    url: "https://bbp-nexus.epfl.ch/staging/v0/data/somatosensorycortexproject/electrophysiology/experimentaltraceset/v0.0.1?filter=%7B%22op%22%3A%22eq%22%2C%22path%22%3A%22rdf%3Atype%22%2C%22value%22%3A%22nsg%3AConfiguration%22%7D&context=%7B%22nsg%22%3A%20%22https%3A%2F%2Fbbp-nexus.epfl.ch%2Fvocabs%2Fbbp%2Fneurosciencegraph%2Fcore%2Fv0.1.0%2F%22%7D",
```
Comment (mdupont) [inline scripts/neuron-pipe/consts.js]: Why the staging URL here (bbp-nexus) while you reference the prod URL (bbp.epfl.ch/nexus) in the files above?

File: scripts/neuron-pipe/helpers.js [hunk 2]
```diff
@@ -27,6 +27,8 @@ function buildURI(base, uriParts, options = {}) {
       return `${prev}/${current}`;
     }, base);
   const params = queryString.stringify(options);
+  console.log({options})
```
Comment (mdupont) [inline scripts/neuron-pipe/helpers.js]: 🚫 console.log()

File: scripts/neuron-pipe/pushToNexus.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ export default async (doc, token, queryURL) => {
         break;
       default:
         console.log(error, responsePayload, updateURL, status);
-        throw new Error("error updating");
+        // throw new Error("error updating");
```
Comment (mdupont) [inline scripts/neuron-pipe/pushToNexus.js]: 🚫  Commented code

File: scripts/neuron-pipe/resources/pctc.js [hunk 4]
```diff
@@ -9,7 +9,17 @@ import pc from "../../testData/pc.json";
...
   let [base, ...urlParts] = getURIPartsFromNexusURL(url);
-  let [error, docs] = await to(getResources(url, token));
+  const options = {
+    filter: JSON.stringify({
+      op: "eq",
+      path: "rdf:type",
+      value: "nsg:Configuration"
+    }),
+    context: JSON.stringify({
+      nsg: "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/"
```
Comment (mdupont) [inline scripts/neuron-pipe/resources/pctc.js]: URL is neither staging or prod?

File: scripts/neuron-pipe/resources/rwbc.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ async function fetch(resource, token, shouldUpload, resourceURL) {
         }
         return doc;
       },
-      downloadMorph(token, short),
+      // downloadMorph(token, short),
```
Comment (mdupont) [inline scripts/neuron-pipe/resources/rwbc.js]: commented code

File: scripts/neuron-pipe/consts.js (hunk 1 above)
Comment (kenjinp) [reply scripts/neuron-pipe/consts.js]: That's where the data is currently living before we ETL it over to nexus v1

File: scripts/neuron-pipe/resources/pctc.js (hunk 4 above)
Comment (kenjinp) [reply scripts/neuron-pipe/resources/pctc.js]: Another example of where the data is living at the moment, we need to fetch it using the right context that lives there

**Pull request merged**